dashboard/app/views/programming_expressions/edit.html.haml

```diff
@@ -0,0 +1,4 @@
+%script{src: webpack_asset_path('js/programming_expressions/edit.js'),
+  data: {programmingExpression: @programming_expression.summarize_for_edit.to_json}}
+= form_for @programming_expression do |f|
```

Comment:
Yeah that's why I used both the form here and ajax below. I originally tried to use fetch but couldn't get the CSRF token in the header. I can try again but I also couldn't find an example of where this was done successfully.

Reply:
okay I'm having some luck using fetch instead of ajax so I'll keep pulling on that thread. I think I tried to use fetch before adding this form in, which is why I thought it didn't work

Reply:
Well, I have no idea why it wasn't working last week but I'm using fetch and it seems to work without `form_for`. Thanks for pointing this out Maddie!